utils/release-checklist

```diff
@@ -3,7 +3,7 @@ Release checklist
 
 * open https://travis-ci.org/sphinx-doc/sphinx/branches and check stable branch is green
 * Check `git status`
-* Run `make style-check`
+* Run `tox`, which will run style checks, type checks, unit tests and doc tests
```

Comment:
Really? Did you mean our release manager should run all tests on local before release?
All tests takes very long time (see Travis, it takes a hour!).